backend/src/main/scala/bloop/reporter/DefaultReporterFormat.scala

```diff
@@ -8,6 +8,11 @@ import scala.compat.Platform.EOL
 object DefaultReporterFormat extends (ConfigurableReporter => ReporterFormat) {
   override def apply(reporter: ConfigurableReporter): DefaultReporterFormat =
     new DefaultReporterFormat(reporter)
+
+  def toOption[T](m: java.util.Optional[T]): Option[T] = if (m.isPresent) Some(m.get) else None
+  def position(position: xsbti.Position): Option[(Int, Int)] = {
```

Comment:
What is `(Int, Int)`? Is it start/end offsets? Or line/column?

Reply:
line/column

Reply:
I would introduce a case class for this instead of using tuple of ints. I would also make it clear if lines and columns start at 0 or 1

Reply:
Yup, all this area needs to be improved when I’ve got some time, so it will be most likely introduced in a future PR